maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/RepositoryManagerDriver.java

```diff
@@ -257,13 +256,19 @@ private void setupBuildRepos(BuildExecution execution, Indy indy)
             }
 
             Group buildGroup = new Group(MAVEN_PKG_KEY, buildContentId);
-            buildGroup.setDescription(String.format("Aggregation group for PNC build #%s", id));
+            String adjective = "";
+            if (snapshotBuild) {
+                adjective = "snapshot ";
+            } else if (tempBuild) {
+                adjective = "temporary ";
+            }
```

Comment:
We don't have anything like snapshotBuild, we just have a temporary builds with, which has less restrictions as for e.g. snapshots are allowed. So, I'd not distinguish it in the name as it will apply to all temporary builds

Reply:
This is just to make the repo description more specific. Why not to do so?

Reply:
OK, it doesn't hurt, so whatever, but we don't have the differentiation of snapshot and tempBuild, so tempBuild = snapshotBuild in this case, once the changes above are applied